src/definitions/modules/checkbox.js

```diff
@@ -239,18 +239,26 @@ $.fn.checkbox = function(parameters) {
               }
             }
 
+            shortcutPressed = false;
             if(key == keyCode.escape) {
               module.verbose('Escape key pressed blurring field');
               $input.blur();
               shortcutPressed = true;
             }
-            else if(!event.ctrlKey && ( key == keyCode.space || (key == keyCode.enter && settings.enableEnterKey)) ) {
-              module.verbose('Enter/space key pressed, toggling checkbox');
-              module.toggle();
-              shortcutPressed = true;
-            }
-            else {
-              shortcutPressed = false;
+            else if(!event.ctrlKey && module.can.change()) {
```

Comment:
`module.can.change` is added to fix some issue?

Reply:
No, i added this, because any possible change change inside the if statement can only happen when the checkbox is not disabled/reachonly (which can.change checks). 
The toggle function tests this itself, but the checked/unchecked dont, so i decided to check this in the first place